### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by ddysher**
**What this PR does / why we need it**:

R.T

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,57 @@
...
 
 [Cyclone Roadmap](./docs/ROADMAP.md)
 
-## History
+## Contributing
+
+If you are interested in contributing to Cyclone, please checkout [CONTRIBUTING.md](./CONTRIBUTING.md).
+We welcome any code or non-code contribution!
 
-Before v0.9.0, Cyclone focuses on CI/CD pipelines and run pipelines in both Docker and Kubernetes.
```
Comment (ddysher) [inline README.md]: we can add this to our cyclone v2 proposal

File: README.md [hunk 2]
```diff
@@ -1,48 +1,57 @@
...
-- **Triggers**: Workflow can be triggered by time schedule, webhooks.
-- **Controllable**: Execution of workflow can be paused, resumed or cancelled.
-- **Multi-Tenant**: Cyclone supports multi-tenant.
-- **Garbadge Collection**: Cleanup after workflow finished and results collected.
+
+- DAG graph scheduling: cyclone supports DAG workflow execution
+- Parameterization: stage (unit of execution) can be parameterized to maximize configuration reuse
+- External dependencies: external systems like SCM, docker registry, S3 can be easily integrated with cyclone
+- Triggers: cyclone supports cron and webhook trigger today, with upcoming support for other types of triggers
+- controllability: workflow execution can be paused, resumed, retried or cancelled
```
Comment (cd1989) [inline README.md]: `Controllability `

**Comment by ddysher** [comment]
@cd1989 @supereagle updated

**Pull request merged**